recipes/playwright/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "playwright" %}
+{% set version = "1.11.0a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kumaraditya303/playwright-python/archive/refs/tags/v1.11.0.a3.tar.gz
```

I don't think your recipe will be accepted without using a legit source, like non-fork github - although I know the latest release is not working. But you should at least use the `version` variable to enforce homogeneous version.

```suggestion
  url: https://github.com/microsoft/playwright-python/archive/v{{ version }}.tar.gz
```

---

@fcollonval I know that but that wouldn't work because the last released version of it at Microsoft repo does not contains many fixes for installation directly from repo See https://github.com/microsoft/playwright-python/pull/537 I am waiting for a new release and then I would change it to official, Thanks for your suggestion